tutorials/Machine_Translation_using_PyTorch_Ignite.ipynb

```diff
@@ -0,0 +1,2295 @@
+   "outputs": [
+    {
+     "output_type": "stream",
+     "name": "stdout",
+     "text": [
+      "Example of a Datapoint \n",
+      "\n",
+      "{'id': '123598', 'translation': {'de': 'Nachrichtenberichte und \u201eAnalysen\u201c der staatlich kontrollierten Sender in Russland und Georgien, die ein negatives Image \u201edes Feindes\u201c zeichnen, dienen lediglich dazu, die Kluft zwischen den ethnischen Gruppen noch zu vertiefen.', 'en': 'News reports and \u201canalysis\u201d by state-controlled channels in both Russia and Georgia that promote negative images of \u201cthe enemy\u201d serve only to widen the gap between ethnic groups.'}} \n",
+      "\n",
+      "Lengths\n",
+      "\t Train Set - 156207\n",
+      "\t Val Set - 66946\n"
+     ]
+    }
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "2SOKRE0KnTMF"
+   },
+   "source": [
+    "## Tokenizer\n",
+    "\n",
+    "The tokenizer needs to be defined to convert the input from strings to token ids. The Machine Translation tokenizers need additional parameters about the source language and target language, refer [here](https://huggingface.co/transformers/model_doc/mbart.html#transformers.MBartTokenizer) for more info."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "metadata": {
+    "colab": {
+     "base_uri": "https://localhost:8080/",
+     "height": 113,
+     "referenced_widgets": [
+      "77359c62be444223903308ffe194dad4",
+      "b5ba9b6b0fd94a75a6d939fba9c0719b",
+      "8609be07594747239694611c9992c58c",
+      "333772c0030a41d09b3d33a11853befe",
+      "88619b6c54974f738fb2dcb8387ff621",
+      "e64297a53a994a419ab8390606a3aed8",
+      "cb7ff9f1e1094109b82c16d2402cfa71",
+      "e6d75c4de2264e4ea1a2314c844aca1f",
+      "e459bcc467c04ab192869a5ed2b6decb",
+      "3c636a4b36434cc4a72d85a957c222e2",
+      "bbf09cdba4114916b0d06a2dbf84132a",
+      "78fb6727c04a41bd9d141713c35e9e3a",
+      "11c70980088d4f1e83b4a88c2f5ed822",
+      "1c4c6dd8741a4f82a2b46f2c90b6729b",
+      "36ee1ee5c8804a2e8f8ec5ad75442fcf",
+      "149c63466ede4ada80b5cd3cf611fd11",
+      "f1469d410f9a432daf586887554def9d",
+      "ffeff12882e444aca8222a78d56494b3",
+      "dc477e655ada4ceea845b040c5f58a38",
+      "ebbb2d0b60724ecab566a771e8533741",
+      "67f508f302b34fd1bc6c54412bf30209",
+      "75fc98a93c2448c89aa38923c3aeafac",
+      "4baa01c483e94ae9b2d2eeced8813b66",
+      "1117f12b0ce64c19a5a66a21a4f0a582",
+      "87c97d0bccb54437a223aa7205a00793",
+      "ec0d36565d18449083407acabbf4eca4",
+      "4e02018a81994e7291369b9e4d72907f",
+      "ca6c0cbf329a44f6abe1497410eae2dd",
+      "dbb21863a68a4274b237cf861372c829",
+      "2fa54423e61148eb8fa5a0172cc7de24",
+      "e25a58d0482146fc8e9bf2093b30a45b",
+      "9a7ee58beed54d7a9599d35f1ef71c06",
+      "307baa6d3c3442a386e8b5f9d3eb68a6"
+     ]
+    },
+    "id": "p-o1azyZVQnW",
+    "outputId": "1f25bd6a-ace9-4d7f-e14f-12377a76ddcd"
+   },
+   "source": [
+    "tokenizer = MBartTokenizer.from_pretrained(\n",
```

Comment:
I have the following error on Colab
```
ImportError: 
MBartTokenizer requires the SentencePiece library but it was not found in your environment. Checkout the instructions on the
installation page of its repo: https://github.com/google/sentencepiece#installation and follow the ones
that match your environment.
```

Reply:
Missing 
```
pip install sentencepiece
```

Reply:
Weird, my colab version has it but somehow its not present here. I have added now, thanks for the reviews.